pkg/reconciler/certificate/certificate.go

```diff
@@ -184,3 +190,29 @@ func (c *Reconciler) updateStatus(desired *v1alpha1.Certificate) (*v1alpha1.Cert
 
 	return c.ServingClientSet.NetworkingV1alpha1().Certificates(existing.Namespace).UpdateStatus(existing)
 }
+
+func (c *Reconciler) setHTTP01Challenges(knCert *v1alpha1.Certificate) {
+	challenges := []v1alpha1.HTTP01Challenge{}
+	for _, dnsName := range knCert.Spec.DNSNames {
+		domainHash := fmt.Sprintf("%d", adler32.Checksum([]byte(dnsName)))
```

Comment:
just use itoa/FormatInt.
Unless you need `%x`, which I am not sure if you need or not.